internal/server/v1/page/golden/bio_page/chlamydia.json

```diff
@@ -2,14978 +2,6 @@
   "nodes": [
     {
       "neighbors": [
-        {
```

Comment:
@spiekos : Is these deletions expected ? Many of the [DiseaseGeneAssociation](https://datacommons.org/browser/DiseaseGeneAssociation) nodes seems to be deleted in the autopush version.